homeassistant/components/lovelace/dashboard.py

```diff
@@ -240,9 +240,10 @@ async def _async_load_data(self) -> collection.SerializedStorageCollection | Non
         updated = False
 
         for item in data["items"] or []:
-            if "-" not in item[CONF_URL_PATH]:
-                updated = True
-                item[CONF_URL_PATH] = f"lovelace-{item[CONF_URL_PATH]}"
+            url_path = item[CONF_URL_PATH]
+            if url_path == "map" or "-" in url_path:
+                continue
+            item[CONF_URL_PATH] = f"lovelace-{url_path}"
```

Comment:
`updated` is always false now.

Reply:
We can remove this check because all user dashboards has been migrated. (line 248 save the data)